### DISCUSSION TITLE
Missing start icon on WQHD screen on Windows 11

### DISCUSSION THREAD

**Discussion by musa0502** [pubpub-zz/ppInk#270] [Q&A] [answered]
Unfortunately, there is no start icon on my Yoga notebook with WQHD screen running Windows 11. What can I do to use ppink on this machine?

**Comment by pubpub-zz** [comment, marked as answer]
First ensure you have applied: this tip:

> Tips
> 
>     There is a known issue for multiple displays of unmatched DPI settings (100%, 125%, 150%, etc.). If you use ppInk on a computer with multiple displays of unmatched DPI settings, or you encounter problems such as incorrect snapshot position, unable to drag toolbar to locations etc., please do the following as a workaround (in Windows 10 version 1903 as an example): right-click ppInk.exe, Properties, Compatibility, Change high DPI settings, Enable override high DPI scaling behavior scaling performed by: Application.
> 

go to  notification area, you should see the ppink Icon, right click and open the options, check then uncheck `show floating point window`
![image](https://github.com/pubpub-zz/ppInk/assets/4083478/bf9cb6cc-2fd3-41a4-b5fa-03ccfbfd1a94)

the floating icon should appear at 100,100 (about left-top corner on the main screen
you can then move it with right click drag and drop then in the options save the new position(`save floating window`) & then `save  to files`

**Reply by musa0502** [reply]
The advice with the dpi high resolution settings helped me. Thanks for that. Now, I can use ppink again.

**Comment by dkettmann** [comment]
Excellent! Was going crazy on this issue.  Program works amazingly on my laptop but went to use on my desktop with 4k TV as secondary display and everything was hidden/all over the place. Even after I disabled that 2nd display was still not any better until I enable the compat settings as described here.  Now it is great again!
